File: 151-reverse-words-in-a-string/reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {
        String str = s.trim();
        String[] arr = str.split(" ");
        // System.out.println(Arrays.toString(arr));
        String add = "";
        for (int i = arr.length - 1; i >= 0; i--) {
            if (arr[i] != "") {
                add += arr[i];

                add += " ";

            }

            // System.out.print(add);

        }
        return add.trim();

    }
}

File: 151-reverse-words-in-a-string/reverse-words-in-a-string.cs
public class Solution {
    public string ReverseWords(string s) {
        String[] arr =s.Split(" ",StringSplitOptions.RemoveEmptyEntries);
        String str="";
        for(int i=arr.Length-1;i>0;i--){
            if(arr[i]!=""){
                   str+=arr[i]+" ";

            }
         


        }
        return str+arr[0];
    }
}